hack/generate-manifest-windows.sh

```diff
@@ -112,6 +118,8 @@ pushd $TMP_DIR > /dev/null
 BASE=../../default
 if [ "$RUNTIME" == "containerd" ]; then
     BASE=../../containerd
+elif [ "$RUNTIME" == "ovs-containerd" ]; then
```

Comment:
Do you forget to add in Makefile like this,
```
	$(CURDIR)/hack/generate-manifest-windows.sh --mode dev --ovs-containerd > build/yamls/antrea-windows-ovs-containerd.yml
```

Reply:
Thanks for reminding this.